Automattic/Gravatar-SDK-iOS
# Make ImageRating enum pure swift

This PR renames `GravatarRating` to `ImageRating`. I thought about naming it `Rating` but in insulation `ImageRating` is easier to understand. 

We also make it pure Swift enum.

Updates to WPUI: https://github.com/wordpress-mobile/WordPressUI-iOS/pull/148

To test:
- Run the demo app
- Go to Image Download
- Set a Rating in the Rating field
  - Check that the returned URL in console has the rating option selected.

closes #82
